src/applications/burials/config/chapters/01-claimant-information/personalInformation.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import fullSchemaBurials from 'vets-json-schema/dist/21P-530V2-schema.json';
+import {
+  fullNameUI,
+  ssnUI,
+  dateOfBirthUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import PrefillMessage from '../../../components/PrefillMessage';
+import { generateTitle } from '../../../utils/helpers';
+
+const {
+  claimantFullName,
+  claimantSocialSecurityNumber,
+  claimantDateOfBirth,
+  formV2,
+} = fullSchemaBurials.properties;
+
+export default {
+  uiSchema: {
+    'ui:title': generateTitle('Personal information'),
+    'ui:description': PrefillMessage,
+    claimantFullName: fullNameUI(),
+    claimantSocialSecurityNumber: {
+      ...ssnUI('Social Security number'),
+      'ui:required': form =>
+        form?.relationshipToVeteran !== 'executor' &&
+        form?.relationshipToVeteran !== 'funeralDirector' &&
+        form?.relationshipToVeteran !== 'otherFamily',
+    },
+    claimantDateOfBirth: {
+      ...dateOfBirthUI('Date of birth'),
+      'ui:required': form =>
+        form?.relationshipToVeteran !== 'executor' &&
+        form?.relationshipToVeteran !== 'funeralDirector' &&
+        form?.relationshipToVeteran !== 'otherFamily',
+    },
+
+    // <------ Needed for version control on the backend ------>
+    // TODO: Once Burials V2 is 100% rolled out, we can remove this and remove the def from vets-json-schema
```

Comment:
@DLarson-Oddball 38-48 needs to be removed per our slack conversation

Reply:
@DLarson-Oddball disregard this. After further discussion we are opting to leave this in temporarily.